src/raglite/_eval.py

```diff
@@ -26,9 +26,7 @@ class QuestionResponse(BaseModel):
         """A specific question about the content of a set of document contexts."""
 
         question: str = Field(
-            ...,
-            description="A specific question about the content of a set of document contexts.",
-            min_length=1,
```

Comment:
Are you removing this because it's not supported in 'strict' mode by OpenAI?

Reply:
yes, exactly.